deps/openssl/openssl_no_asm.gypi

```diff
@@ -13,9 +13,9 @@
       'includes': ['config/archs/linux-ppc64/no-asm/openssl.gypi'],
     }, 'target_arch=="s390x" and OS=="linux"', {
       'includes': ['config/archs/linux64-s390x/no-asm/openssl.gypi'],
-    }, 'target_arch=="arm" and OS=="linux"', {
+    }, 'target_arch=="arm" and OS in "linux android"', {
       'includes': ['config/archs/linux-armv4/no-asm/openssl.gypi'],
-    }, 'target_arch=="arm64" and OS=="linux"', {
+    }, 'target_arch=="arm64" and OS in "linux android"', {
```

Comment:
`"linux android"` —> `("linux", "android")` because `nux` and `droid` are in the former but not in the latter.

Reply:
thanks, 
i will update, and could you please explain what the difference?

Reply:
```
>>> for x in ("linux", "android", "nux", "and", "droid"):
...     print(x in ("linux android"), x in ("linux", "android"))
...
True True
True True
True False
True False
True False
```

Reply:
Got it, thanks